### ISSUE TITLE
Fingerprint loaded with `--nofp`

### ISSUE THREAD

**Issue by lemmy** [tlaplus/tlapm#42]
```shell
markus@banana:~$ tlapm --nofp BlockingQueue.tla
(* loading fingerprints in "BlockingQueue.tlaps/fingerprints" *)

...

(* created new "BlockingQueue.tlaps/BlockingQueue.thy" *)
(* fingerprints written in "BlockingQueue.tlaps/fingerprints" *)

markus@banana:~$ tlapm --version
1.4.5 (build 33809)
```

**Comment by johnyf** [comment]
Thank you for reporting this issue. This message is printed from:

https://github.com/tlaplus/tlapm/blob/ab564e3563b7d530b33bf0022de1f999a28a74d7/src/tlapm.ml#L345

and when the command-line option `--nofp` is given, this message could be changed to something like:

"(* will not use fingerprints (because of option `--nofp`), but will now load fingerprints from file `...` in order to overwrite with the new fingerprints, and then save the results at the end. *)"

Another relevant change would be to rephrase the description of option `--nofp` in `tlapm`'s help message (what is printed by `tlapm --help`):

https://github.com/tlaplus/tlapm/blob/ab564e3563b7d530b33bf0022de1f999a28a74d7/src/tlapm_args.ml#L198

based on this comment:

https://github.com/tlaplus/tlapm/blob/ab564e3563b7d530b33bf0022de1f999a28a74d7/src/params.ml#L34

The value `!Params.no_fp` is in all of `tlapm` used at one place:

https://github.com/tlaplus/tlapm/blob/ab564e3563b7d530b33bf0022de1f999a28a74d7/src/backend/prep.ml#L589-L594

This documentation change will also emphasize the difference with the option `--cleanfp`:

https://github.com/tlaplus/tlapm/blob/ab564e3563b7d530b33bf0022de1f999a28a74d7/src/tlapm_args.ml#L201-L202

Sidenote: For a file with about the same proof obligations as in the earlier run of `tlapm` that generated the existing fingerprints file, it seems that the difference between the options `--nofp` and `--cleanfp` is small, when the options `--toolbox` and `--line` are not given. When the option `--toolbox` specifies a subset of lines in the file, and when the option `--line` is given, then it seems to me that there is a larger difference between `--nofp` and `--cleanfp`.

**Comment by johnyf** [comment]
Addressed by commit 4b4b5adbfc2bf0d6f8c530a61b70c36d52d91391.
